scl/sumologic/sumologic.conf

```diff
@@ -0,0 +1,79 @@
+#############################################################################
+# Copyright (c) 2020 Balabit
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+
+# EXAMPLES:
+#
+# Just send all syslog to sumologic:
+#    log {
+#        source { system(); };
+#        destination { sumologic(token("USER-TOKEN-AS-PROVIDED-BY-sumologic")); };
+#    };
+#
+# With TLS encryption (make sure you trust the sumologic CA cert by putting it
+# to /etc/ssl, or create a separate CA directory):
+#
+#    log {
+#       destination {
+#                sumologic(token("USER-TOKEN-AS-PROVIDED-BY-sumologic") port(6514) transport(tls)
+#                       tls(peer-verify(required-trusted) ca-dir('/etc/ssl/certs'))
+#                );
+#       };
+#    };
+#
+#
+# Send JSON data:
+#    log {
+#        source { system(); };
+#        destination {
+#            sumologic(token("USER-TOKEN-AS-PROVIDED-BY-sumologic")
+#                   template("$(format-json --scope all-nv-pairs)"));
+#            );
+#        };
+#    };
+#
+#
+# Send already parsed apache logs to sumologic:
+#    log {
+#        source { file("/var/log/apache2/access.log" flags(no-parse)); };
+#        parser { apache-accesslog-parser(); };
+#        destination {
+#            sumologic(token("USER-TOKEN-AS-PROVIDED-BY-sumologic")
+#                   tag(apache)
+#                   template("$(format-json .apache.* timestamp=${ISODATE})"));
+#        };
+#    }
+#
+
+@requires json-plugin
+
+block destination sumologic(token(TOKEN)
+                         tag("tag")
+                         host('syslog.collection.ca.sumologic.com')
```

Comment:
If I understand right only the `ca` part is the variable here. If that's right, it might be more convenient to define a `deployment()` option, and assemble the full `host` like:
```
"syslog.collection.`deployment`.sumologic.com"
```

Reply:
Done